src/storage/rdb.cc

```diff
@@ -157,7 +157,7 @@ StatusOr<std::string> RDB::loadEncodedString() {
     unsigned char buf[4] = {0};
     if (len == RDBEncInt8) {
       auto next = GET_OR_RET(stream_->ReadByte());
-      return std::to_string(static_cast<int>(next));
+      return std::to_string(static_cast<int8_t>(next));
```

Comment:
Why `int` if the length is 8?

Reply:
Did you try `uint8_t`?

Reply:
It should be correct to use int8_t here since the encoding is int8. Cast it to `int` will drop the sign bit.